tutorials/05-linear-regression/05_linear-regression.jmd

```diff
@@ -75,12 +74,14 @@ train_target = trainset[:, target]
 test_target = testset[:, target]
 
 # Standardize the features.
-μ, σ = rescale!(train; obsdim=1)
-rescale!(test, μ, σ; obsdim=1)
+dt_features = fit(ZScoreTransform, train; dims=1)
+StatsBase.transform!(dt_features, train)
+StatsBase.transform!(dt_features, test)
 
 # Standardize the targets.
-μtarget, σtarget = rescale!(train_target; obsdim=1)
-rescale!(test_target, μtarget, σtarget; obsdim=1);
+dt_targets = fit(ZScoreTransform, train_target)
```

Comment:
Personally a bit curious about the difference between `MLDataUtils.rescale!` and `StatsBase.fit and transform`: are they producing the same results (i.e. mean and std)? In R and Python they are the same, e.g. in Scikit-learn pre-processing module it's called scalers (normal, min-max, etc) with methods `fit`, `transform` or `fit_transform`.

Reply:
There are many more alternwtives (eg in TableTransforms or FeatureTransforms) but StatsBase is sufficient here and a indirect dependency anyway. It yields the same result as the previous version with the deprecated MLDataUtils (this functionality was not and will not be transferred to MLUtils), but the version in StatsBase only works with vectors and matrices (which is sufficient here).